api/backend.go

```diff
@@ -758,3 +758,28 @@ func (b *StatusBackend) UpdateMailservers(enodes []string) error {
 	}
 	return st.UpdateMailservers(nodes)
 }
+
+// SignHash exposes vanilla ECDSA signing for signing a message for Swarm
+func (b *StatusBackend) SignHash(hexEncodedHash string) (string, error) {
+	hash, err := hexutil.Decode(hexEncodedHash)
+
```

Comment:
So many spaces between each call and error handling :D

Reply:
I love whitespaces!

Reply:
adressed